.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -48,16 +48,12 @@ Thank you!
 Describe changes from the user side, and list all potential break changes or other risks.
 --->
 
+- Use a developer-oriented tone and narrative style without detailing the fix.
+- Describe the issue and its impact on developers.
+- Describe the user’s first-hand experience of the issue, not your resolution approach.
+- Refer to: ant.design/changelog-cn
```

Comment:
```suggestion
- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: ant.design/changelog
```